scripts/docker-entrypoint.sh

```diff
@@ -15,26 +15,28 @@ map_uidgid() {
 }
 
 set_permissions() {
-    # Set permissions for consumption directory
-    chgrp paperless "$PAPERLESS_CONSUMPTION_DIR" || {
-        echo "Changing group of consumption directory:"
-        echo "  $PAPERLESS_CONSUMPTION_DIR"
-        echo "failed."
-        echo ""
-        echo "Either try to set it on your host-mounted directory"
-        echo "directly, or make sure that the directory has \`o+x\`"
-        echo "permissions and the files in it at least \`o+r\`."
-    } >&2
-    chmod g+x "$PAPERLESS_CONSUMPTION_DIR" || {
-        echo "Changing group permissions of consumption directory:"
-        echo "  $PAPERLESS_CONSUMPTION_DIR"
-        echo "failed."
-        echo ""
-        echo "Either try to set it on your host-mounted directory"
-        echo "directly, or make sure that the directory has \`o+x\`"
-        echo "permissions and the files in it at least \`o+r\`."
-    } >&2
-
+    # Set permissions for consumption and export directory
+    for i in PAPERLESS_CONSUMPTION_DIR PAPERLESS_EXPORT_DIR; do
+      cur_dir_name=$(sed -e's/PAPERLESS_//; s/_DIR//' <<< $i | tr '[:upper:]' '[:lower:]')
```

Comment:
While I wouldn't mind the message being "Changing group of PAPERLESS_CONSUMPTION_DIR", I appreciate the effort to get a normal sentence.

Three things (one I see as required, the other two optional):

1. *Required:* Please add a comment before the variable explaining what this does -- it took me a hot minute to understand what is going on. Maybe something like:

       # Extract the name of the current directory for the error message

2. *Optional:* I would change the loop-variable from `$i` to something more speaking, e.g. `$dir`. `$i` is usually used for integer variables.

3. *Optional:* I would favour using `awk` instead of both `sed` and `tr`, mainly because I think it enhances readability:

       cur_dir_name=$(echo "$i" | awk -F'_' '{ print tolower($2); }')

   But this is probably subjective, and with the added comment mentioned in (1), feel free to keep it as is! 👍